Rebus.AdoNet/Dialects/PostgreSqlDialect.cs

```diff
@@ -46,6 +46,7 @@ public PostgreSqlDialect()
 
 		#region Overrides
 		public override bool SupportsSelectForUpdate => true;
+		public override bool SupportsPgTryAdvisoryLock => true;
```

Comment:
Properties indicating support for specific SQL sintax/features should go into Dialect base class (returning false or the appropiate default value). And then such properties are overridden on derived Dialect classes like this..

Reply:
Btw, rename to SupportsTryAdvisoryLockFunction